lms/djangoapps/discussion/tasks.py

```diff
@@ -93,6 +93,7 @@ def send_ace_message(context):  # lint-amnesty, pylint: disable=missing-function
 @shared_task(base=LoggedTask)
 @set_code_owner_attribute
 def send_ace_message_for_reported_content(context):  # lint-amnesty, pylint: disable=missing-function-docstring
+    log.info('Reported content email notification task started.')
```

Comment:
This log also does not seem very useful

Reply:
This log indicates the `START` of the task!
We will compare this to the previous log to see the time difference in between!